domains/vu.json

```diff
@@ -0,0 +1,11 @@
+{
+  "description": "Vu-is-Me",
+  "repo": "https://github.com/vuisme/",
+  "owner": {
+    "username": "vuisme",
+    "email": "me@vutn.net"
+  },
+  "record": {
+    "A": "140.245.62.158"
```

Comment:
404 not found, could you send a preview? Also just place it within an array.

Reply:
Hi,
It’s my cluster ip. After point to this i will make ingress tls challenge late